dashboards-reports/server/routes/utils/constants.ts

```diff
@@ -87,6 +79,25 @@ export const EXTRA_HEADERS = [
   'x-forwarded-for',
 ];
 
+export const converter = new Showdown.Converter({
+  tables: true,
+  simplifiedAutoLink: true,
+  strikethrough: true,
+  tasklists: true,
+  noHeaderId: true,
+});
+
+export const replaceBlockedKeywords = (htmlString: string) => {
+  htmlString = htmlString.replace('iframe', 'BLOCKED_KEYWORD');
+  htmlString = htmlString.replace('localhost', 'BLOCKED_KEYWORD');
+  htmlString = htmlString.replace('127.0.0.1', 'BLOCKED_KEYWORD');
+  htmlString = htmlString.replace('0.0.0.0', 'BLOCKED_KEYWORD');
+  htmlString = htmlString.replace('0:', 'BLOCKED_KEYWORD');
+  htmlString = htmlString.replace('169.254.169.254', 'BLOCKED_KEYWORD');
+  htmlString = htmlString.replace('[fd00:ec2::254]', 'BLOCKED_KEYWORD');
```

Comment:
i think this only replaces the first occurrence, use replaceAll?

Reply:
good catch